src/PointwiseFunctions/Hydro/Units.hpp

```diff
@@ -0,0 +1,41 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+namespace hydro {
+/*!
+ * \brief Functions, constants, and classes for converting between different
+ * units.
+ *
+ * In SpECTRE we prefer to use geometric units where \f$G=c=M_{\odot}=1\f$, as
+ * is standard in numerical relativity. However, in order to interface with
+ * other codes, we need to sometimes convert units. This namespace is designed
+ * to hold the various conversion factors and functions. A unit library/system
+ * would be nice, but is a non-trivial amount of work.
+ */
+namespace units {
+/*!
+ * \brief Entities for converting between geometric units where
+ * \f$G=c=M_{\odot}=1\f$ and CGS units.
+ */
+namespace cgs {
+/// `mass_cgs = mass_geometric * mass_unit`
+constexpr double mass_unit = 1.0 / 5.02765209e-34;
+/// `length_cgs = length_geometric * length_unit`
+constexpr double length_unit = 1.0 / 6.77140812e-6;
+/// `time_cgs = time_geometric * time_unit`
+constexpr double time_unit = 1.0 / 2.03001708e5;
+/// Baryon mass, used to go from number density to rest mass
+/// density. I.e. `rho_cgs = n_cgs * baryon_mass`, where `n_gcs` is the number
+/// density in CGS units.
+constexpr double baryon_mass = 1.659e-24;
```

Comment:
Is it possible to use a flexible baryon mass? Not all EOSs use the same one, so that quantity could be misinterpreted. (I just recently had that issue confusing atomic mass unit for neutron mass...)

Reply:
Hmm, do you think we should just not define this globally then? Since the other quantities are accepted physical constants they seem fine to have globally, but since the baryon mass is a choice people tweak, maybe that we should just leave to the individual EOS?

Reply:
Ideally, you would need this to be stored in two places. In the EOS, to know what baryon mass its compatible with. That would give you the correct number density (schematically n_baryon = rho/eos::baryon_mass;)
However, you ID code might assume that you are using a different baryon mass. e.g. Lorene assumes that you always have atomic mass units. So you might want to rescale things in the exporter, to make sure that you get the same
rho = n_export * exporter::baryon_mass density.

Reply:
Okay, I will hard-code the mass into the exporter and file an issue that we need to figure out how to do baryon mass for the EOS. Is it that each EOS would define a member function `baryon_mass`? I guess for a polytrope this could depend on `K`? Or is this something we can hard-code to e.g. the value here? Then I guess for tables we would need to get it from compose, right?

Does that sound like a reasonable plan?

Reply:
Sounds great! :)  Also let me add, the only time this matters as far as I'm aware is when using neutrino interactions.
So for a polytrope you probably wouldn't need this, or just define a baryon mass, as long as it's used consistently that should be fine. The value here would probably be perfect for everything that doesn't have its on baryon mass.

Yep, most tables specify it, or in the case of compose the internal energy comes just rescaled to the neutron mass. (That value is currently hardcoded in the table import PR, too).

Reply:
Okay, great! Thank you :D